recipes-graphics/xorg-lib/libxkbcommon_0.8.0.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Generic XKB keymap library"
+DESCRIPTION = "libxkbcommon is a keymap compiler and support library which \
+processes a reduced subset of keymaps as defined by the XKB specification."
+HOMEPAGE = "http://www.xkbcommon.org"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e525ed9809e1f8a07cf4bce8b09e8b87"
+LICENSE = "MIT & MIT-style"
+
+DEPENDS = "util-macros flex-native bison-native"
+
+SRCREV = "b82e3b764e60df337ca695e8f8642e7bf42b0cca"
+SRC_URI = "git://github.com/xkbcommon/libxkbcommon.git;protocol=https"
```

Comment:
Why are you copying the whole recipe to meta-nilrt, instead of using a bbappend?

Reply:
`SRC_URI[md5sum]`, `SRC_URI[sha256sum]`, `UPSTREAM_CHECK_URI ` are set in the original recipe which we don't want. It seemed cleaner to override the recipe instead of deleting these in bbappend.

Reply:
I don't think I agree. It is a fairly common pattern for bbappends to update those variables. And preferring a bbappend will let the meta-nilrt code more clearly communicate what parts from upstream we're *intending* to change (the SRC_URI) - which helps maintainability.

Reply:
Ok, updated it to be a bbappend.